### DISCUSSION TITLE
Beneficios no disponibles

### DISCUSSION THREAD

**Discussion by QuinoSanz** [community/community#76745] [GitHub Education]
### Hi everyone,

Ayer envie mi solicitud para github education y fue aprobada pero no dispongo de los beneficios. Si entro a revisar cual es mi plan de github me sale plan Free tal y como se ve en la foto 
![image](https://github.com/community/community/assets/136028272/70877378-823b-4e1d-a066-22c5e5090392)

Ademas, a la hora de usar VSCode el copilot no me reconoce la cuenta como una cuenta suscrita al plan
![image](https://github.com/community/community/assets/136028272/c0714aff-e047-4043-ab51-e8088a9cfebc)

Sin embargo mi cuenta si que ha sido aprobada
![image](https://github.com/community/community/assets/136028272/b43fe21f-07ac-4ccd-8365-641c053df7c4)

**Comment by Akash1134** [comment]
Hi @QuinoSanz , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
